python-sdk/docs/development/DEVELOPMENT.md

```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
 
 * `dev/dags/` - DAG Files
 * `dev/logs/` - Logs files of the Airflow containers
+
+
+## Adding support for a new database
+
+You can use Test Driven approach for adding support for a new database to the Astro Python SDK. You can fulfil the
+[tests](https://github.com/astronomer/astro-sdk/tree/main/python-sdk/tests) for all the
+Python SDK operators by adding parameters to the existing tests for the database and additionally adding more database specific
+implementation tests. You can take a look at this [PR](https://github.com/astronomer/astro-sdk/pull/639) and
+[PR](https://github.com/astronomer/astro-sdk/pull/753) on how to add parameters for the existing tests for all the
+operators.
+
+To start with you can take the following steps for the initial configuration:
+1. Add the database name constant to the [Database](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/src/astro/constants.py#L42) class in the `constants.py` module.
```

Comment:
If we point to specific lines we should reference specific commits - not the main branch, which is constantly moving.

Reply:
makes sense, done, thanks.